fmcg_store/data/product_attribute_value.xml

```diff
@@ -0,0 +1,99 @@
+        <field name="name">288.75g</field>
+        <field name="attribute_id" ref="product_attribute_1"/>
+    </record>
+    <record id="product_attribute_value_13" model="product.attribute.value">
+        <field name="name">481.25 g</field>
+        <field name="attribute_id" ref="product_attribute_1"/>
+    </record>
+    <record id="product_attribute_value_16" model="product.attribute.value">
+        <field name="name">600gram - PACK of 4</field>
+        <field name="attribute_id" ref="product_attribute_1"/>
+    </record>
+    <record id="product_attribute_value_17" model="product.attribute.value">
+        <field name="name">300gram - PACK of 2</field>
+        <field name="attribute_id" ref="product_attribute_1"/>
+    </record>
+    <record id="product_attribute_value_18" model="product.attribute.value">
+        <field name="name">195g</field>
+        <field name="attribute_id" ref="product_attribute_1"/>
+    </record>
+    <record id="product_attribute_value_19" model="product.attribute.value">
+        <field name="name">350g</field>
+        <field name="attribute_id" ref="product_attribute_1"/>
+    </record>
+    <record id="product_attribute_value_21" model="product.attribute.value">
+        <field name="name">70g Pouch</field>
+        <field name="attribute_id" ref="product_attribute_5"/>
+    </record>
+    <record id="product_attribute_value_22" model="product.attribute.value">
+        <field name="name">140g Pouch</field>
+        <field name="attribute_id" ref="product_attribute_5"/>
+    </record>
+    <record id="product_attribute_value_23" model="product.attribute.value">
+        <field name="name">280g Pouch</field>
+        <field name="attribute_id" ref="product_attribute_5"/>
+    </record>
+    <record id="product_attribute_value_32" model="product.attribute.value">
+        <field name="name">200g</field>
+        <field name="attribute_id" ref="product_attribute_1"/>
+    </record>
+    <record id="product_attribute_value_33" model="product.attribute.value">
+        <field name="name">300g</field>
+        <field name="attribute_id" ref="product_attribute_1"/>
+    </record>
+    <record id="product_attribute_value_34" model="product.attribute.value">
+        <field name="name">600g - Pack of 3</field>
+        <field name="attribute_id" ref="product_attribute_5"/>
+    </record>
+    <record id="product_attribute_value_35" model="product.attribute.value">
+        <field name="name">Pack of 4</field>
+        <field name="attribute_id" ref="product_attribute_5"/>
+    </record>
+    <record id="product_attribute_value_36" model="product.attribute.value">
+        <field name="name">Pack of 1</field>
+        <field name="attribute_id" ref="product_attribute_5"/>
+    </record>
+    <record id="product_attribute_value_37" model="product.attribute.value">
+        <field name="name">Pack of 5</field>
+        <field name="attribute_id" ref="product_attribute_5"/>
+    </record>
```

Comment:
As discussed, "pack of ..." means that we should use the product.packaging model